docs/team/thing1thing2.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: John Doe's Project Portfolio Page
+title: Thing1Thing2's Project Portfolio Page
```

Comment:
I think you might need to use your actual name VS ur github handle for PPP

Reply:
Noted, thank you 👍